mam/v2/mam/message.c

```diff
@@ -808,7 +811,12 @@ retcode_t mam_msg_recv_packet(mam_msg_recv_context_t *ctx, trits_t *b,
   tryte_t checksum = -1;
 
   /*  absorb long trint ord; */
-  ERR_BIND_RETURN(pb3_unwrap_absorb_longtrint(&ctx->spongos, b, &ctx->ord), e);
+  {
+    trit_t ord_trits[18];
```

Comment:
Can we have a this block in a function? (code duplication)

Reply:
Will probably disappear as part of #901